Code/Framework/AzFramework/AzFramework/Archive/ZipDirCacheFactory.cpp

```diff
@@ -60,7 +60,7 @@ namespace AZ::IO::ZipDir
     {
         m_szFilename = szFileName;
 
-        CachePtr pCache{ new Cache{ &AZ::AllocatorInstance<AZ::OSAllocator>::Get()} };
+        CachePtr pCache{ new Cache{} };
```

Comment:
Should this use `aznew`?

Reply:
That's what I figured, but I only mentioned it because there seemed to be a lot of `aznew`/`azfree`/`azmalloc` in these changes.